apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/dao/mapper/DexTradeEntryMapper.java

```diff
@@ -18,8 +18,8 @@
     
     @Override
     public DexTradeEntry map(ResultSet rs, StatementContext ctx) throws SQLException {
-        DexTradeEntry dexTradeEntry = new DexTradeEntry();
-        dexTradeEntry.setDbId(rs.getLong("db_id"));
+        DexTradeEntry dexTradeEntry = new DexTradeEntry(rs);
```

Comment:
I think, that better to create Derived table for DexTradeEntry and then inherit from DerivedEntityMapper.

Reply:
You are not right. 
1. Latest column is not mandatory for derived table, latest required only for versioned derived tables.
2. For current impl, dex_trade should be derived to support blockchain rollback
3. Its an exchange bug, not yours, but better to fix it entirely here

Reply:
Sorry, but .... 
Task is about exporting exchange data, not about fixing bug of exchange.
If fixing that bug needs time, I can wait and later update the code accordingly.

Reply:
As agreed on meeting, need to remove dex_trade table/classes entirely